pkg/sysinfo/sysinfo.go

```diff
@@ -5,10 +5,10 @@ package sysinfo
 type SysInfo struct {
 	AppArmor bool
```

Comment:
It seems `golint` will not get crazy about that, but I agree with you it is better to add comments on those exported fields.